docs/adding_packages/dependencies.md

```diff
@@ -0,0 +1,20 @@
+# Dependencies
+
+This section outlines all the practices and guidelines for `requirements()` and `build_requirements()` methods. This includes everything for "vendored" dependencies to
+when and how the versions could be changes.
+
+<!-- toc -->
+## Contents
+
+  * [Rules](#rules)<!-- endToc -->
+
+## Rules
+
+* Version range is not allowed.
```

Comment:
```suggestion
* [Version range](https://docs.conan.io/en/latest/versioning/version_ranges.html) is not allowed.
```
in general, try to put links to the relevant docs, whenever possible. newcomers might not be familiar with all the conan concepts and terminologies we use here.